src/app/components/select-field/index.html

```diff
@@ -63,6 +63,13 @@
     >
     Specifies the selection mode that determines whether users can select one item or multiple items in the select field component's modal. The valid options are <sky-code>single</sky-code>, which displays a text input and allows users to select one item, and <sky-code>multiple</sky-code>, which displays a button and allows users to select multiple items.
     </sky-demo-page-property>
+    <sky-demo-page-property
```

Comment:
We also need to document the new `addNewRecordButtonClick` event. In the PR where the input was added, the output is at https://github.com/blackbaud/skyux-select-field/pull/30/files#diff-d6e48fc07613709745cf9d2b62b7fa31R120.